_dashboards/visbuilder.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: VisBuilder
+nav_order: 8
+---
+
+# VisBuilder
+
+VisBuilder is an experimental feature and shouldn't be used in a production environment. For updates on its progress, or if you want to leave feedback that helps improve the feature, see the [git issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2280).
+{: .warning}
+
+You can use the VisBuilder visualization type in OpenSearch Dashboards to create data visualizations easily using a drag-and-drop gesture. With VisBuilder you have:
+
+* an immediate view of your data without the need to preselect the visualization output
+* the flexibility to change visualization types and index patterns quickly
+* the ability to easily navigate between multiple screens
+
+<img src="{{site.url}}{{site.baseurl}}/images/vis-builder-2.png" alt="VisBuilder new visualization start page">
+
+## Try VisBuilder in the OpenSearch Dashboards playground
+
+If you'd like to try out VisBuilder without installing OpenSearch locally, you can do so in the [Dashboards playground](https://playground.opensearch.org/app/vis-builder#/). VisBuilder is enabled by default.
+
+## Try VisBuilder locally
+
+The VisBuilder feature flag, `vis_builder.enabled:`, is enabled by default in the `opensearch_dashboards.yml` file. 
```

Comment:
If VisBuilder has only one feature flag, named `vis_builder.enabled`, then the commas are appropriate, but if there are several feature flags, of which `vis_builder.enabled` is one, then no commas.

Reply:
It has only one feature flag.